### DISCUSSION TITLE
Anchor link focus will not change to the anchorlink. Change is only visual.

### DISCUSSION THREAD

**Discussion by Onni-Vilhelm** [elementor/elementor#25145] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Fixing the accessibility problem where focus is not changed when clicking an anchor link. If we make an anchor link to some page, the links works visually OK, but is should also change the focus to the section it scrolls to. This is an accessibility problem.

### Describe the solution you'd like.

There is already a working element with the same idea. This is the "skip to content" link in the Hello Elementor theme, which works correctly by skipping the menu and going to main content and moving also the focus there. Anchor link should be done the same way. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by quantizor** [comment]
There is no explicit roadmap. styled-components is in maintenance mode — the library will continue to receive bugfixes and minor improvements, but no major new features are planned.

For context, see the announcement: https://opencollective.com/styled-components/updates/thank-you
